sentry/src/transports/thread.rs

```diff
@@ -7,6 +7,7 @@ use std::time::Duration;
 use super::ratelimit::{RateLimiter, RateLimitingCategory};
 use crate::{sentry_debug, Envelope};
 
+#[expect(clippy::large_enum_variant)]
 enum Task {
     SendEnvelope(Envelope),
```

Comment:
I thought I had mentioned this before, but cannot find the comment.

I believe boxing the Envelope should allow us to avoid this [`large_enum_variant`](https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant) warning. 

```suggestion
enum Task {
    SendEnvelope(Box<Envelope>),
```